src/test/system_tests/ceph_s3_tests/test_ceph_nsfs_s3_config_and_run_s3_tests.sh

```diff
@@ -0,0 +1,17 @@
+# This script objective is to run Ceph s3 tests
+# We use it inside of K8S job, see noobaa-tests-s3-job
+# since we use "|| true" this job will always end with "Completed" status
```

Comment:
So how will we know if the test has actually failed or passed?

Reply:
Makes sense. Then I think we should get rid of both `|| true`.

Reply:
^ @shirady @v-harihar

Reply:
Yes, I agree